platforms/core-runtime/internal-instrumentation-api/src/main/java/org/gradle/internal/instrumentation/api/annotations/UpgradedProperty.java

```diff
@@ -43,6 +45,25 @@
 
     UpgradedAccessor[] originalAccessors() default {};
 
+    BinaryCompatibility binaryCompatibility() default METHODS_REMOVED;
```

Comment:
❓ WDYT about calling these "accessors" instead of "methods"? We already use `originalAccessors` and `UpgradedAccessor`.

Reply:
Makes sense, renamed this and bunch of things from "methods" to "accessors".